raml-maven-plugin/src/test/resources/validator/no-raml/pom.xml

```diff
@@ -0,0 +1,28 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>uk.gov.justice.raml.test</groupId>
+    <artifactId>project-to-test</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>raml-maven-plugin</artifactId>
+                <groupId>uk.gov.justice.maven</groupId>
+                <configuration>
+                    <sourceDirectory>src/raml</sourceDirectory>
```

Comment:
Is it worth making src/raml a pom property - in the parent ? - If you agree, this also applied to the other pom files above.